libheif/heif_cxx.h

```diff
@@ -284,7 +284,7 @@ namespace heif {
 
     bool is_full_range() const;
 
-    void set_color_primaties(heif_color_primaries cp);
+    void set_color_primaries(heif_color_primaries cp);
```

Comment:
As noted, I could include the old name as a deprecated version.

C++ classes are brittle which makes things interesting. In some ways, it might make more sense to just force people to update when they upgrade. But, I'm not sure what the repository's policy is for C++...